packages/ember-glimmer/lib/component-managers/custom.ts

```diff
@@ -52,125 +139,129 @@ export interface ComponentArguments<T = {}> {
   * `update()` - invoked when the arguments passed to a component change
   * `getContext()` - returns the object that should be
 */
-export default class CustomComponentManager<T> extends AbstractComponentManager<
-  CustomComponentState<T> | null,
-  DefinitionState
-> {
-  constructor(private delegate: CustomComponentManagerDelegate<T>) {
+export default class CustomComponentManager<ComponentInstance = Opaque>
+  extends AbstractComponentManager<CustomComponentState<ComponentInstance>, DefinitionState>
+  implements
+    WithStaticLayout<
+      CustomComponentState<ComponentInstance>,
+      DefinitionState<ComponentInstance>,
+      OwnedTemplateMeta,
+      RuntimeResolver
+    > {
+  constructor(private delegate: ManagerDelegate<ComponentInstance>) {
     super();
   }
 
   create(
     _env: Environment,
     definition: DefinitionState,
     args: Arguments,
-    dynamicScope: DynamicScope
-  ): CustomComponentState<T> {
+    _dynamicScope: DynamicScope
```

Comment:
if we don't need it we can just remove the arg right?

(where did we land w.r.t. view tracking again?)

Reply:
Confirm, will remove.

We landed on: `TODO: Make this work with the Ember Inspector` 😝